fbw-a380x/src/wasm/systems/a380_systems/src/fuel/fuel_quantity_management_system.rs

```diff
@@ -308,78 +308,76 @@ impl RefuelApplication {
     }
 
     fn lookup_trim_fuel_from_target_fuel_range(
-        &mut self,
-        zfw_category: &HashMap<u32, Vec<u32>>,
-        total_fuel_desired_rounded: u32,
+        target_zfw_cg_keys: &[u32],
+        target_fuel_range: &[u32],
         zero_fuel_weight_cg_rounded: u32,
     ) -> Mass {
-        let target_fuel_range = zfw_category.get(&total_fuel_desired_rounded).unwrap();
-
-        let cg_index = self
-            .trim_tank_map
-            .params
-            .target_zfw_cg
+        let cg_index = target_zfw_cg_keys
             .iter()
             .position(|&x| x == zero_fuel_weight_cg_rounded)
             .unwrap_or_default();
         Mass::new::<kilogram>(target_fuel_range[cg_index] as f64)
     }
 
-    fn lookup_trim_fuel_from_mapping(
-        &mut self,
-        zfw_category: &HashMap<u32, Vec<u32>>,
+    fn normalise_fuel_range(
+        min_fuel_desired: u32,
+        max_fuel_desired: u32,
         total_desired_fuel_rounded: u32,
-        zero_fuel_weight_cg_rounded: u32,
-    ) -> Mass {
-        let lowest_fuel_desired = *(zfw_category.keys().min().unwrap());
-        let largest_fuel_desired = *(zfw_category.keys().max().unwrap());
-
+    ) -> u32 {
         match total_desired_fuel_rounded {
-            x if x < lowest_fuel_desired => self.lookup_trim_fuel_from_target_fuel_range(
-                zfw_category,
-                lowest_fuel_desired,
-                zero_fuel_weight_cg_rounded,
-            ),
-            x if x > largest_fuel_desired => self.lookup_trim_fuel_from_target_fuel_range(
-                zfw_category,
-                largest_fuel_desired,
-                zero_fuel_weight_cg_rounded,
-            ),
-            _ => self.lookup_trim_fuel_from_target_fuel_range(
-                zfw_category,
-                total_desired_fuel_rounded,
-                zero_fuel_weight_cg_rounded,
-            ),
+            x if x < min_fuel_desired => min_fuel_desired,
+            x if x > max_fuel_desired => max_fuel_desired,
+            _ => total_desired_fuel_rounded,
```

Comment:
Isn't this the same as this?:
```suggestion
            total_desired_fuel_rounded.clamp(min_fuel_desired, max_fuel_desired )
```

Reply:
true, this static function may additionally no longer be needed.